app/controllers/geckoboard_controller.rb

```diff
@@ -24,4 +24,11 @@ def confirmed_bookings
             ]
     }
   end
+
+  private
+
+  def leaderboard_report
+    LeaderboardReport.new(0.95, ApplicationHelper.
+                                  instance_method(:prison_name_for_id))
```

Comment:
My personal preference for things like this would be:

```ruby
LeaderboardReport.new(
  0.95,
  ApplicationHelper.instance_method(:prison_name_for_id)
)
```

(I don't really like lining up indentation as it results in volatile changesets.)